packages/myst-cli/src/build/init.ts

```diff
@@ -38,6 +38,17 @@ const SITE_CONFIG = `site:
   domains: []
 `;
 
+const GITIGNORE = `/_build/
+
+# Python development
+.ipynb_checkpoints/
+__pycache__/
```

Comment:
This is the simplest list I could think of for Python-based projects. We could even drop the Python entries, and just ignore `_build`?

Reply:
Yeah, I think we should be super simple. Just `_build`!